test/common/upstream/health_check_fuzz.cc

```diff
@@ -1,28 +1,100 @@
 #include "test/common/upstream/health_check_fuzz.h"
 
 #include <chrono>
+#include <memory>
+
+#include "common/grpc/common.h"
 
 #include "test/common/upstream/utility.h"
 #include "test/fuzz/utility.h"
 
 namespace Envoy {
 namespace Upstream {
+namespace { // gRPC helper methods
+// From unit tests
+std::vector<uint8_t>
+serializeResponse(grpc::health::v1::HealthCheckResponse::ServingStatus status) {
```

Comment:
was this used as a member before?
if so, the cleanest way is to re-use the member function, and only have the `convertToGrpcStatus` converting from the proto test::common::upstream::ServingStatus to the grpc::health::v1::HealthCheckResponse::ServingStatus enum

Reply:
Will move around

Reply:
Changed!